docker/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Dockerfile for universalresolver/driver-did-dns
 
-FROM maven:3-jdk-11 AS build
+FROM maven:3-openjdk-17-slim AS build
```

Comment:
Should we maybe use this also instead of `3-eclipse-temurin-17-focal` here? https://github.com/decentralized-identity/universal-resolver/blob/main/uni-resolver-web/docker/Dockerfile#L3C12-L3C38